datasets/simulated.py

```diff
@@ -41,79 +41,119 @@ def __init__(
         img_size=32,
         n_classes=2,
         train_frac=0.8,
-        framework='pytorch',
+        val_frac=0.8,
```

Comment:
This name is inconsistent, should be `0.2` and using `1-val_frac` for `n_train` no?